Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -3315,12 +3315,24 @@ def com_google_fonts_check_040(ttFont, vmetrics):
 
 @check(
   id = 'com.google.fonts/check/042'
```

Comment:
we need to add a comma at the end of this line

Reply:
This is why I don't write any code, lmao